### DISCUSSION TITLE
How to share to Mastodon ?

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5005] [Q&A] [answered]
Hi,
I feel dumb but I can't see how to share an item from my freshrss reading list to Mastodon.
In Settings > share I've filled in the name and URL of my Mastodon account but when I click on "share" I receive an error  "page not found" :
`"https://mastodon.top/@myaccount/share?title=Comment%20int%C3%A9grer%20des%20notes%20de%20bas%20de%20page%20accessibles%20en%20HTML%20%3F%20%C2%B7%20Access42&url=https%3A%2F%2Faccess42.net%2Fcomment-integrer-notes-bas-page-accessibles`

I've seen that I can initiate an app in Mastodon but then where do I add the keys and token  in freshrss ?

Sharing with shaarli or email works OK (freshRSS and sharli are on the same server).

Thanks.

**Comment by Frenzie** [comment]
Leave out the `@myaccount`.

**Comment by math-GH** [comment, marked as answer]
Here my example:
![grafik](https://user-images.githubusercontent.com/1645099/211649882-724456ef-fcf4-4317-af0a-18557652ddf6.png)

**Reply by scharmach** [reply]
To avoid confusion, should the URL field then perhaps be called "Instance URL" instead?

**Reply by math-GH** [reply]
for Mastodon it sounds valid. But it is general text string that should fit to all other sharing services too.

**Comment by ghost** [comment]
So simple... thanks
